sample-apps/react/e2ee-demo/src/App.css

```
@@ -0,0 +1,45 @@
+:root {
+  --color-bg: #0f1218;
+  --color-surface: #1a1f2b;
+  --color-surface-elevated: #242b3a;
+  --color-border: #2d3548;
+  --color-text: #e8eaed;
+  --color-text-secondary: #8b95a8;
+  --color-text-mono: #7dd3fc;
+  --radius-md: 8px;
+  --radius-sm: 4px;
+  --font-mono: 'SF Mono', 'Cascadia Code', 'Fira Code', Consolas, monospace;
```

Stylelint (17.7.0)

[error] Expected "Consolas" to be "consolas" (value-keyword-case)

(value-keyword-case)